multiqc/modules/mosdepth/mosdepth.py

```diff
@@ -12,6 +12,8 @@
 from multiqc.modules.qualimap.QM_BamQC import coverage_histogram_helptext, genome_fraction_helptext
 from multiqc.plots import linegraph, bargraph
 
+import fnmatch
```

Comment:
Nitpick: Imports are a bit messy here already, but anyway if you could move next to `import logging` that would be nice 👍🏻